drivers/hmis/app/graphql/mutations/submit_form.rb

```diff
@@ -126,6 +129,11 @@ def resolve(input:)
           enrollment_id: enrollment&.EnrollmentID,
           personal_id: enrollment&.PersonalID,
         }
+      when 'Hmis::File'
+        {
+          client_id: Hmis::Hud::Client.viewable_by(current_user).find_by(id: input.client_id)&.id,
+          enrollment_id: Hmis::Hud::Enrollment.viewable_by(current_user).find_by(id: input.enrollment_id)&.id,
```

Comment:
This isn't used with the current approach, right? Since Enrollment ID is collected _as part of the form_, I would expect Enrollment ID to get set on the file during the `FileProcessor` evaluation.

I think it's good to leave 135 here anyway, because we might want to add an "Upload File" button on the View Enrollment page in the future. With that, the Enrollment would be passed as input (as opposed to collected in the form).